docs/2-installation.md

```diff
@@ -41,9 +42,14 @@ Then add the following commands to the `scripts` field in the package.json in yo
 
 ## Advanced
```

Comment:
This section can probably be extracted to a "deploy" chapter later on.

Reply:
Yeah. Definetly.